pom.xml

```diff
@@ -115,8 +115,8 @@
     <felix-framework-version>5.6.12</felix-framework-version>
 
     <site-repo-url>scpexe://people.apache.org/www/activemq.apache.org/maven/</site-repo-url>
-    <source-version>1.8</source-version>
-    <target-version>1.8</target-version>
+    <source-version>8</source-version>
```

Comment:
Forgive my ignorance, but why do we set this source-version so low?

Reply:
I am guessing either just because no 11-level source features are actually being used yet, or alternatively perhaps some things actually start failing to compile if it is bumped, even if they do work when its already targetting 11, because the compiler it starts enforcing some things it didnt while source level remains set to 8. (That happened with the changes I did for Artemis, 8+11 worked, 11+11 needed changes).

Reply:
@gemmellr yep. Setting to source-version 11 causes the module export errors to show up, since the build hasn't set the module exports yet.